src/Collapsable.jsx

```diff
@@ -0,0 +1,99 @@
+var React = require('react');
+var joinClasses = require('./utils/joinClasses');
+var BootstrapMixin = require('./BootstrapMixin');
+var CollapsableMixin = require('./CollapsableMixin');
+var classSet = require('./utils/classSet');
+var domUtils = require('./utils/domUtils');
+var cloneWithProps = require('./utils/cloneWithProps');
+
+var ValidComponentChildren = require('./utils/ValidComponentChildren');
+var createChainedFunction = require('./utils/createChainedFunction');
+
+
+var Collapsable = React.createClass({
```

Comment:
The mixin, `CollapsableMixin` was spelled Collapsable and I rolled with it. So I would propose renaming that as well. The name `CollapsibleNav` makes sense to me. Would you like another PR with the proposed changes or are you taking it from here?